src/routes/formsgSiteCreation.ts

```diff
@@ -82,20 +81,20 @@ export class FormsgRouter {
       if (!siteName) {
         const err = `A site name is required`
         await this.sendCreateError(requesterEmail, repoName, submissionId, err)
-        return res.sendStatus(200)
+        return cloneRes.sendStatus(200)
```

Comment:
heads up that the types mismatch here !

Reply:
yea due to vscode renaming the whole `res` stack instead of the var